assets/js/search.js

```diff
@@ -114,9 +114,13 @@ function search (nodes, value) {
   $oldContent.hide()
   $content.append($results)
 
-  function closeResults (event) {
-    if (event.metaKey) {
-      return
+  function closeResults (e) {
+    var event = e || window.event
+    if (typeof event !== 'undefined' && typeof event === 'object' && event !== null) {
```

Comment:
well, we I guess we can get rid of `typeof event !== 'undefined'`, 
but we need the rest because.

```js
> typeof null
"object"
```